tests/test_all.py

```diff
@@ -150,10 +153,37 @@ def test_zypper_dup_works(container_per_test: ContainerData) -> None:
 
     container_per_test.connection.run_expect(
         [0],
-        f"timeout 2m zypper -n dup --from {repo_name} -l -d -D "
+        f"timeout 2m zypper -n dup --from {repo_name} -l "
```

Comment:
```suggestion
        f"timeout 5m zypper -n dup --from {repo_name} -l "
```

Since we are here. `aarch64` is sometimes really slow.

Reply:
done!